packages/testing-kit/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@safe-global/testing-kit",
+  "version": "1.0.0",
+  "description": "Helper package providing testing utilities",
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
+  "keywords": [
+    "Ethereum",
+    "Wallet",
+    "Safe",
+    "Smart Account",
+    "SDK",
+    "Testing"
+  ],
+  "scripts": {
+    "format:check": "prettier --check \"*/**/*.{js,json,md,ts}\"",
+    "format": "prettier --write \"*/**/*.{js,json,md,ts}\"",
+    "unbuild": "rimraf dist artifacts deployments cache .nyc_output *.tsbuildinfo",
+    "build": "yarn unbuild && hardhat compile && NODE_OPTIONS=--max-old-space-size=8192 tsc -p tsconfig.build.json && tsc-alias -p tsconfig.build.json && yarn copy-files && chmod +x dist/bin/testing-kit.js",
+    "copy-files": "cpx 'artifacts/**/*' dist/artifacts",
```

Comment:
Why do you need to copy them to this route?

Reply:
A thing of the initial iterations that remained there. Good catch